lib/src/services/preferences.dart

```diff
@@ -15,6 +15,9 @@ class Preferences extends Service {
 
 	late SharedPreferences _prefs;
 
+	///
+	static String refreshData(String data) => "refresh_$data";
```

Comment:
I'd say to remove this function. It doesn't say `getRefreshData`, but it implies that it actually does so, when really all it does it return a string. You can probably just hard-code this string template in the actual function.

```suggestion
```

Reply:
but if we eventually add another form of data that needs to be updated isn't it just easier to just leave this?

Reply:
See how I have it in my suggestion below. The `"refresh_$data"` part is good, because it lets you use a variable to create a unique string. I'm just saying it doesn't need to be its own function, and you can stick it into the below function.